scio-core/src/main/scala/com/spotify/scio/JavaConverters.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Spotify AB.
```

Comment:
Wrong year. You can use `$today.year` substitution in IntelliJ's copyright settings.